apps/challenges/admin.py

```diff
@@ -45,3 +51,10 @@ class LeaderboardDataAdmin(TimeStampedAdmin):
     list_display = ("challenge_phase_split", "submission", "leaderboard", "result")
     list_filter = ("challenge_phase_split", "submission", "leaderboard", "result")
     search_fields = ("challenge_phase_split", "submission", "leaderboard", "result")
+
+
+@admin.register(ChallengeConfiguration)
+class ChallengeConfigurationAdmin(TimeStampedAdmin):
+    list_display = ("user", "challenge", "zip_configuration", "is_created", "stdout_file", "stderr_file",)
+    list_filter = ("user", "challenge", "is_created",)
+    search_fields = ("user", "challenge",)
```

Comment:
@taranjeet I think keeping `challenge` in search_fields will give us the flexibility to search for the zip file of that challenge and we can remove it from list_filter. What do you say?